TelCo.ColorCoder/GetColor.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Diagnostics;
+using System.Drawing;
+using static TelCo.ColorCoder.MajorMinorColors;
+using static TelCo.ColorCoder.ColorPair;
+
+namespace TelCo.ColorCoder
+{
+    class GetColor
```

Comment:
class name should be noun, function name verb
ColorGetter to start with, can think of better name